internal/executor/doctor/executor.go

```diff
@@ -153,11 +160,16 @@ func (d *Executor) Help(context.Context) (api.Message, error) {
 func (d *Executor) getGptClient(cfg *Config) (gpt3.Client, error) {
 	d.l.Lock()
 	defer d.l.Unlock()
-	if cfg.ApiKey == "" {
+	if cfg.APIKey == "" {
 		return nil, fmt.Errorf("OpenAPI API Key cannot be empty. You generate it here: https://platform.openai.com/account/api-keys")
 	}
+
+	var opts []gpt3.ClientOption
+	if cfg.APIBaseURL != "" {
+		opts = append(opts, gpt3.WithBaseURL(cfg.APIBaseURL))
+	}
```

Comment:
I didn't test that but as mentioned, it looks like it should work. However, to be on the safe side please add 2 more options:
- `WithOrg`
- `WithDefaultEngine`

because I'm afraid that if the different base URL will be used, the engine name may be also different or some org id param will be required.